committing.rst

```diff
@@ -454,34 +336,19 @@ Developers can apply labels to GitHub pull requests).
 .. _cherry_picker.py: https://github.com/python/core-workflow/tree/master/cherry_picker
 
 
-.. _forking:
-
-Forking repository
-------------------
-
-Forking a repository on GitHub is as simple as clicking Fork button in the right
-upper corner at https://github.com/python/cpython.
-
+Reverting a Merged Pull Request
+-------------------------------
 
-Maintaining a repository
```

Comment:
After asking around during the sprint, the consensus was to remove this section.